include/mixer.h

```diff
@@ -52,6 +52,13 @@ struct AudioFrame {
 	float right = 0;
 };
 
+typedef int16_t MixerSample_t;
```

Comment:
Mixes camel and snake case (sorry for not catching that earlier!)

Because it's not a struct or class, `mixer_sample_t` should do.